Challenges/Enumerable/code blocks Chp 5/map_methods.rb

```diff
@@ -0,0 +1,15 @@
+scores = {low: 2, high: 8, avg: 6}
+
+adjusted_scores = scores.map do |k,v|
+"#{k.capitalize}: #{v * 100}"
+end 
```

Comment:
(A few people on our team like to use the ⛏️ emoji to indicate small nitpicks we have)

⛏️ For this kind of operation where you can fit the whole thing on one line, we'll usually use the one line syntax

```ruby
adjusted_scores = scores.map { |k, v| "#{k.capitalize}: #{v * 100}" }
```

Reply:
OK, so when possible. 
use one line syntax. I'm sure that would like neater (if that is a word). Thank you